app/src/assets/localization/en/protocol_command_text.json

```
@@ -12,6 +13,7 @@
   "degrees_c": "{{temp}}°C",
   "disengaging_magnetic_module": "Disengaging Magnetic Module",
   "dispense": "Dispensing {{volume}} µL into well {{well_name}} of {{labware}} in {{labware_location}} at {{flow_rate}} µL/sec",
+  "dispense_push_out": "Dispensing {{volume}} µL into well {{well_name}} of {{labware}} in {{labware_location}} at {{flow_rate}} µL/sec with push out volume {{push_out_volume}} µL",
```

@ecormany and this is the commandText for dispensing with the pushOut param

---

oof this is long but i dunno that there's much we can do…
```suggestion
  "dispense_push_out": "Dispensing {{volume}} µL into well {{well_name}} of {{labware}} in {{labware_location}} at {{flow_rate}} µL/sec and pushing out {{push_out_volume}} µL",
```
do we need to state the push-out flow rate or is it constant?